idd/Energy+.idd.in

```diff
@@ -44175,15 +44175,21 @@ ZoneHVAC:LowTemperatureRadiant:VariableFlow,
         \key OnePerSurface
         \key CalculateFromCircuitLength
         \default OnePerSurface
-   N16; \field Circuit Length
+   N16, \field Circuit Length
         \units m
         \default 106.7
+   A18; \field Changeover Delay Time Period Schedule
```

Comment:
This doesn't explicitly say it is required, though you do have "minimum fields" incremented up one.  Perhaps we don't need to increment min-fields after all?

Reply:
@Myoldmopar Yeah, I'm still a bit fuzzy on that (you'd think I'd have it down after all these years).  Technically, it isn't required and the code just assumes no changeover delay if the schedule isn't there.  So, I guess it really isn't required and I should change the min-fields field?